src/Boids.hs

```diff
@@ -51,9 +54,14 @@ guideBoidToPoint :: Point -> Boid -> Boid
 guideBoidToPoint guidePoint boid = guideBoidToVector direction boid
     where direction = fromPoints (boidPosition boid) guidePoint
 
+
+isWithinView :: Boid -> Boid -> Bool
```

Comment:
`isWithinViewOf` might be a better name because it makes partial application more descriptive: it's perfectly clear what `isWithinViewOf pivotBoid` means. By extension, this should make the full application less confusing: people familiar with Haskell will expect arguments to be ordered in the order of descending frequency of partial application, and that "Of" at the end of the function name will disambiguate what boid comes first—the one that's *looking* or the one that are *looked at*.